packages/base/src/dialogs/symbology/components/color_ramp/ColorRampValueControls.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from '@jupyterlab/ui-components';
+import React from 'react';
+
+import { IColorRampValueControlsProps } from '@/src/types';
+
+export const ColorRampValueControls: React.FC<IColorRampValueControlsProps> = ({
+  min,
+  setMin,
+  max,
+  setMax,
+  rampDef,
+  initialMin,
+  initialMax,
+  renderType,
+}) => {
+  return (
+    <>
+      <div className="jp-gis-symbology-row">
+        <label htmlFor="min-value">Min Value:</label>
+        <input
+          id="min-value"
+          type="number"
+          value={min ?? ''}
+          onChange={e =>
+            setMin(
+              e.target.value !== '' ? parseFloat(e.target.value) : undefined,
+            )
+          }
+          className="jp-mod-styled"
+          placeholder="Enter min value"
+        />
+      </div>
+
+      {rampDef.type === 'Divergent' && renderType === 'Graduated' && (
+        <div className="jp-gis-symbology-row">
+          <label htmlFor="critical-value">Critical Value:</label>
+          <input
+            id="critical-value"
+            type="number"
+            value={rampDef.criticalValue ?? ''}
```

Comment:
Instead of e.g. `0.5` here, if the `selectedMin` is -10 and `selectedMax` is `10`, I'd want to see `0`. The actual data value that will be at the divergence point in the color map.

For example, "Colormap diverges at 50% between min and max data values, which is `0`." Something like that?